src/app/globals.css

```
@@ -180,6 +180,316 @@ nav a.active::after {
   }
 }
 
+/* ============================================
+   Hiero contribution cards
+   ============================================ */
+.hiero-reveal {
+  @apply opacity-0;
+  transform: translateY(1rem);
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)